src/onemod/stage/base.py

```diff
@@ -53,41 +66,105 @@ def skip_if(self) -> set[str]:
         return self._skip_if
 
     @classmethod
-    def from_json(cls, filepath: Path | str, name: str) -> Stage:
-        """Create stage object from JSON file."""
+    def from_json(
+        cls,
+        filepath: Path | str,
+        name: str | None = None,
+        from_pipeline: bool = False,
```

Comment:
Just to make sure I am understanding the use cases here. The use case for this `from_pipeline` argument could be if someone wants to use a single json config for their Pipeline (makes sense), but they need to initialize and/or evaluate some portion of their stages separately. Is this correct? Now that I type it out I think it probably is desirable to have the option.

Reply:
sgtm. I think it makes sense to leave the default as `False`